#7 As an agent, I want to create product

draganculibrk9 opened the pull request:
closes #31, closes #30
resolves #7

draganculibrk9 committed:
#31 Implement product creation on backend

draganculibrk9 committed:
#30 Implement product creation view

draganculibrk9 committed:
Merge branch 'development' into #7

draganculibrk9 committed:
#7 Add missing parenthesis

MasterRadule commented on Backend/src/models/products/product_dto.go:
```diff
@@ -8,3 +13,22 @@ type Product struct {
+	if p.Price <= 0 {
+		return rest_errors.NewBadRequestError("Product price must be greater than zero")
+	}
+	if p.OnStock < 0 {
```
Type of OnStock is uint. This check is surplus.

draganculibrk9 committed:
#7 Make requested changes

Resolves https://github.com/Nistagram-Organization/Agent-Application/pull/39#discussion_r639048622
